Because we still deliver Python 2.7 variant of pylint, we need to have setup.py
available (and cannot use the new pyproject.toml based build).

--- pylint-2.15.0/setup.py
+++ pylint-2.15.0/setup.py
@@ -0,0 +1,3 @@
+from setuptools import setup
+
+setup()
